M2/Macaulay2/d/common.d

```diff
@@ -100,24 +100,12 @@ export setup(e:SymbolClosure,fun1:unop,fun2:unop):void := (
      e.symbol.unary = fun1;
      e.symbol.postfix = fun2;
      );
-export setupop(s:SymbolClosure,fun:unop):void := (
```

Comment:
A very good reason to keep `setupop` is for ease of searching through the codebase for places where operators are declared.